dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -68,4 +70,18 @@ def check_for_unit_completion
       update!(status: COMPLETED)
     end
   end
+
+  private
+  def enroll_in_module(learning_module)
+    return unless learning_module.plc_course_unit == plc_course_unit
```

Comment:
gotcha. I just wanted to be sure we're ok with silently skipping the irrelevant ones.